sdk/identity/identity/review/identity.api.md

```diff
@@ -173,6 +179,7 @@ export interface InteractiveBrowserCredentialCommonOptions extends TokenCredenti
 // @public
 export interface InteractiveBrowserCredentialOptions extends InteractiveBrowserCredentialCommonOptions {
     clientId?: string;
+    tokenCachePersistenceOptions?: TokenCachePersistenceOptions;
```

Comment:
These options should also contain `disableAutomaticAuthentication?: bool` and `authenticationRecord?: AuthenticationRecord`

Reply:
authenticationRecord exists in the common options already 🌞

Reply:
I just added: disableAutomaticAuthentication